api/src/org/labkey/api/exp/PropertyType.java

```
@@ -202,6 +205,53 @@ public Object getPreviewValue(@Nullable String prefix)
             return prefix + "Value";
         }
     },
+    MULTI_CHOICE("http://cpas.fhcrc.org/exp/xml#multiChoice", "MultiChoice", '?' /* unsupported in exp.PropertyValues */, JdbcType.ARRAY, 0, "textarea", CellType.STRING, List.class)
+    {
+        @Override
+        protected Object convertExcelValue(Cell cell) throws ConversionException
+        {
+            return ConvertUtils.convert(cell.getStringCellValue(), MultiChoice.Array.class);
+        }
+
+        @Override
+        public Object convert(Object value) throws ConversionException
+        {
+            if (value instanceof String)
+                return value;
+            else
+                return ConvertUtils.convert(value);
+        }
+
+        @Override
+        public SimpleTypeNames.Enum getXmlBeanType()
+        {
+            return SimpleTypeNames.STRING;
+        }
+
+        @Override
+        protected void init(PropertyRow row, Object value)
+        {
+            row.stringValue = (String)value;

```

Should this throw new UnsupportedOperationException() instead?

---

I forgot to go back to look at all these "side" methods on this enum.  I need to see how these are used.